lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -256,6 +261,8 @@ def build_entity(self):
             name=self._module.params['name'],
             description=self._module.params['description'],
             comment=self._module.params['comment'],
+            import_=True if (self._module.params['state'] == 'import' and storage_type in ['iscsi', 'fcp']) else None,
```

Comment:
sure, let me know if you think it will be better to add it to all type of storage domains